src/App.tsx

```diff
@@ -27,6 +28,8 @@ function App() {
     return (pageName: string) => navigate(`/${pageName.toLowerCase()}`);
   }
 
+  const { darkMode } = useDarkMode();
+  console.log("darkMode: ", darkMode);
```

Comment:
Please remove log statements when merging. Only use it to debug.

Reply:
fixed